config/read.go

```diff
@@ -291,7 +305,7 @@ func getDefaultConfig() *ServerConfig {
 		config.AlwaysCollectSystemData = parseConfigBool(alwaysCollectSystemData)
 	}
 	if disableCitusSchemaStats := os.Getenv("DISABLE_CITUS_SCHEMA_STATS"); disableCitusSchemaStats != "" {
-		config.DisableCitusSchemaStats = parseConfigBool(disableCitusSchemaStats)
+		config.DisableCitusSchemaStats = parseConfigDisableCitusSchemaStats(disableCitusSchemaStats)
```

Comment:
@lfittl while working on #494, I realized that this was not the right way to write this because of how we load config. This is only the default config plus config from env, and would be overridden by anything in the config file, if one is present. This should probably be in `preprocessConfig` instead, right?

Reply:
Ah yeah, good point - I think that's right.